packages/playwright-test/src/index.ts

```diff
@@ -24,6 +24,7 @@ import type { TestInfoImpl } from './worker/testInfo';
 import { rootTestType } from './common/testType';
 import { type ContextReuseMode } from './common/config';
 import { artifactsFolderName } from './isomorphic/folders';
+import { runningTestBody } from './common/globals';
```

Comment:
This looks like a layering violation to me: index.ts is a part of Playwright, globals are a part of folio. Why do we need to know the stage of the test if we are the ones that start and end the tracing?

Reply:
We don't.